### PR TITLE
Print debug information for started/stopped transactions in tests

### PR DESCRIPTION
**Pull request by DavertMik**
Following the proposal from @ThomasLandauer  https://github.com/Codeception/Codeception/issues/4326#issuecomment-309429330
I added debug notifications when transaction is started for tests.

Probably the message should be improved, so suggestions are welcome

### CODE REVIEW

File: src/Codeception/Module/Laravel5.php [hunk 1]
```diff
@@ -175,6 +175,8 @@ public function _before(\Codeception\TestInterface $test)
 
         if ($this->applicationUsesDatabase() && $this->config['cleanup']) {
             $this->app['db']->beginTransaction();
+            $this->debugSection('Transaction', 'Started');
+
```
Comment (Naktibalda) [inline src/Codeception/Module/Laravel5.php]: Please delete this blank line.

**Comment by ThomasLandauer** [comment]
Please commit, so I can see the message "live" :-)

**Comment by Naktibalda** [comment]
@ThomasLandauer checkout `debug-transactions` branch from this repo.

**Comment by ThomasLandauer** [comment]
Well, I switched to this branch, downloaded the ZIP file and copied it manually into `vendor/codeception`. Would there have been an easier/better way?

Anyway, I can't see any difference. Am I looking in the right place? ;-) I expected a new line in the output, somewhere in here:
```
Acceptance Tests (1) ---------------------------------------------------------------------------------------------------
Modules: Symfony, Doctrine2, WebDriver, Asserts, \Helper\Acceptance
------------------------------------------------------------------------------------------------------------------------
Testing acceptance
```

**Comment by Naktibalda** [comment]
Do you run it with `-vv` flag?

**Comment by ThomasLandauer** [comment]
Yes. What would the output be supposed to look like?

**Comment by Naktibalda** [comment]
`[Transaction] Started`

**Comment by ThomasLandauer** [comment]
Well, maybe I didn't see it cause I had `cleanup: false`... ;-)))

Two suggestions:

1. Reword it to: `Database: Transaction startet` and `Database: Transaction stopped; all changes reverted.` Reason: Give more context (i.e. "database") to indicate what sort of "transaction" we are talking about.

1. More important: Give this information in the header line too, namely the `true` as well as the `false` case, like this:
````
Acceptance Tests (1) ---------------------------------------------------------------------------------------------------
Modules: Symfony, Doctrine2, WebDriver, Asserts, \Helper\Acceptance
Doctrine2 is in transaction rollback mode (cleanup: true)
Doctrine2 is in non-transaction mode (cleanup: false)
------------------------------------------------------------------------------------------------------------------------
Testing acceptance
````

**Comment by DavertMik** [comment]
1. OK, I asked for better texting so yes, good suggestion, I will apply
2. No, output becomes too verbose

**Comment by ThomasLandauer** [comment]
Are you kidding? That's just one (!) line more of debug output, which is (probably) several hundreds of lines long anyway!

We could "trade" it for another line, which is superflous (seriously):

```
Codeception PHP Testing Framework v2.3.3
Powered by PHPUnit 5.7.20 by Sebastian Bergmann and contributors.

Functional Tests (26) --------------------------------------------------------------------------------------------------
Testing functional
```
The last line ("Testing functional") should be removed, since it doesn't give any new information.

**Comment by DavertMik** [comment]
> Doctrine2 is in transaction rollback mode (cleanup: true)

Sorry module doesn't have access to output other than debug so it can't print messages like this one.

I changed wording. If everything ok, @Naktibalda please merge it
